### PR TITLE
fix: build script for actions in typescript template[EXT-4509]

### PR DESCRIPTION
**Pull request by vikaskumr**
## Purpose

Introduced a build script to bundle the transpiled code in ESM.

## Approach

Every action will be bundled  in ESM code for the TypeScript template with the help of build script.

### CODE REVIEW

File: examples/hosted-app-action-templates/typescript/build-action.js [hunk 1]
```diff
@@ -0,0 +1,35 @@
+const esbuild = require('esbuild');
+const { resolve } = require('path');
+const yargs = require('yargs/yargs');
```
Comment (kdamball) [inline examples/hosted-app-action-templates/typescript/build-action.js:3]: Is this package already available?

File: examples/hosted-app-action-templates/typescript/build-action.js [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+const { hideBin } = require('yargs/helpers');
```
Comment (kdamball) [inline examples/hosted-app-action-templates/typescript/build-action.js:4]: Same here with the package

**Review by kdamball — COMMENTED** [review]
minor stuff missing but good start

File: examples/hosted-app-action-templates/typescript/build-action.js (hunk 1 above)
Comment (vikaskumr) [reply examples/hosted-app-action-templates/typescript/build-action.js:3]: Yes, when I tested it didn't gave any error regarding this package to add it as a dev dependency

File: examples/hosted-app-action-templates/typescript/build-action.js (hunk 2 above)
Comment (vikaskumr) [reply examples/hosted-app-action-templates/typescript/build-action.js:4]: Yes

**Pull request merged**